provider/go.mod

```diff
@@ -4,12 +4,13 @@ go 1.16
 
 replace (
 	github.com/hashicorp/go-getter => github.com/hashicorp/go-getter v1.4.0
+	github.com/hashicorp/terraform-plugin-sdk/v2 => github.com/pulumi/terraform-plugin-sdk/v2 v2.0.0-20210629210550-59d24255d71f
```

Comment:
Is having a forked provider new?

Reply:
That's not a provider. That's our version of the TF SDK. This dependency is common to all providers.